autocodesign/devportalclient/appstoreconnectclient/profiles_test.go

```diff
@@ -191,3 +190,36 @@ func newResponse(t *testing.T, status int, body map[string]interface{}) *http.Re
 
 	return &resp
 }
+
+func Test_wrapInProfileError(t *testing.T) {
+	tests := []struct {
+		name        string
+		err         error
+		wantErrType error
+	}{
+		{
+			err: appstoreconnect.ErrorResponse{
+				Errors: []appstoreconnect.ErrorResponseError{{}},
+			},
+			wantErrType: appstoreconnect.ErrorResponse{},
+		},
+		{
+			err: &appstoreconnect.ErrorResponse{
+				Response: &http.Response{
+					StatusCode: http.StatusNotFound,
+				},
+			},
+			wantErrType: autocodesign.ProfilesInconsistentError{},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			err := wrapInProfileError(tt.err)
+			if tt.wantErrType == nil {
```

Comment:
Do we need this if logic here if none of the test cases a nil `wantErrType` value? I would remove it and later add it back if it will be ever needed.